lib/active_record/connection_adapters/pedant_mysql2_adapter.rb

```diff
@@ -37,8 +37,7 @@ def initialize(message, code, level, query)
 end
 
 class ActiveRecord::ConnectionAdapters::PedantMysql2Adapter < ActiveRecord::ConnectionAdapters::Mysql2Adapter
-
-  alias_method :original_execute, :execute
+  alias_method :exec_delete_without_pedant, :exec_delete
```

Comment:
Is the alias needed here? Would not `super` work since this method is defined in the superclass?